wordlist

```diff
@@ -429,6 +429,7 @@ virtualized
 vmstat
 vtype
 wildcards
+xdel
```

Comment:
Is this really neeed? How come XADD and the others are not in the wordlist?

Reply:
No longer needed - it was part of xdel-max-id which wasn't quoted, but was renamed anyway.